# sensor name
string label

# current temp (if cpu with more than one core, show the highest core temp) 
float32 current

# high temp value
float32 high

# critical temp value
float32 critical
